lib/statsd/class-statsd.php

```diff
@@ -160,5 +164,7 @@ public function send( $data, $sample_rate = 1 ) {
 			$escaped_url = addslashes( $url );
 			trigger_error( "Statsd::send exception('$escaped_url'): {$e->getMessage()}", \E_USER_WARNING ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 		}
+
+	// phpcs:enable WordPress.PHP.DevelopmentFunctions.error_log_trigger_error
```

Comment:
Best to skip the error code for `phpcs:enable`, so that it's clear that all violations should be reported again after this line, and the reader doesn't have to go reading upwards to see if more than `error_log_trigger_error` was disabled.

Reply:
Fixed in 989b3dc93